### PR TITLE
chore: some review feedback

### PR DESCRIPTION
**Pull request by FinnDore**
Some review feeed back for #1427

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: cli/src/cli/index.ts [hunk 1]
```diff
@@ -163,9 +163,8 @@ export const runCli = async () => {
     if (cliResults.flags.tailwind) cliResults.packages.push("tailwind");
     if (cliResults.flags.prisma) cliResults.packages.push("prisma");
     if (cliResults.flags.nextAuth) cliResults.packages.push("nextAuth");
-    if (cliResults.flags.prettierAndExtendedEslint) {
+    cliResults.flags.prettierAndExtendedEslint =
       cliResults.flags.prettierAndExtendedEslint = true;
```
Comment (juliusmarminge) [inline cli/src/cli/index.ts:167]: ??

Comment (FinnDore) [reply cli/src/cli/index.ts:167]: ughhh

File: cli/template/extras/config/_eslintrc.cjs [hunk 2]
```diff
@@ -19,7 +19,13 @@ const config = {
     project: path.join(__dirname, "tsconfig.json"),
   },
   plugins: ["@typescript-eslint"],
-  extends: ["next/core-web-vitals", "plugin:@typescript-eslint/recommended"],
+  extends: [
+    "next/core-web-vitals",
+    "plugin:@typescript-eslint/recommended",
+    "eslint-config-next",
+    "eslint-config-prettier",
+    "prettier",
```
Comment (juliusmarminge) [inline cli/template/extras/config/_eslintrc.cjs:27]: Doesnt this cause conflicts?

Comment (FinnDore) [reply cli/template/extras/config/_eslintrc.cjs:27]: yeah you might be right. leme start a discussion on the parent pr as im not sure what we are going to go for with the extended eslint config

**Comment by FinnDore** [comment]
Gonna close this one for now so we can rationalise what we want in the parent pr

**Pull request closed without merging**